src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -82,10 +87,27 @@ export default class List extends SfdxCommand {
     }
 
     private showDeviceList(list: any[]) {
+        let duration: number = 0;
+
+        const obs = new PerformanceObserver((items) => {
+            duration = items.getEntries()[0].duration / 1000;
+            performance.clearMarks();
+        });
+        obs.observe({ entryTypes: ['measure'] });
+
+        performance.measure(
```

Comment:
Since I anticipate more performance markers over time, I would like to see us create a more formal definition for them. I think we could create a new module that houses our various perf marker configurations, e.g.:
```typescript
const fetchDevicePerf = PerformanceMarkers.getByName(FETCH_DEVICE_MARKER_NAME);
console.log(`${fetchDevicePerf.name}, ${fetchDevicePerf.startMark}, ${fetchDevicePerf.endMark});
// FetchDevices, StartFetchDevices, EndFetchDevices
```
Just to make sure we're not fat-fingering constants around different modules.

Reply:
We can get crazier with that encapsulation too, if you want—adding a facade to `performance` and tracking the different properties (e.g. duration) in each `PerformanceMarker` class, etc. Up to you. 😀 I primarily just want to make sure we're codifying the configuration points (marker names, etc.).